README.md

```diff
@@ -1 +1,7 @@
-# openpower-hw-isolation
\ No newline at end of file
+# openpower-hw-isolation
+In OpenPOWER based system, a user or application can be isolate hardware and the respective
+isolated hardware will be ignored to init during the next boot of the host.
+
+**Note:** 
+- System must be a power-off state when a user want to isolate hardware.
```

Comment:
The system must be in a power-off state when a user wants to isolate hardware.

Reply:
Will change